changelog.md

```diff
@@ -0,0 +1,44 @@
+# Changelog
+All notable changes to this project will be documented in this file.  
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+##  - 2022 
```

Comment:
@MariaMaximova, I insist on deleting this line as it breaks markdown structure: you have section `Unreleased`, then if finishes and new section `2022` starts. If this line is removed, then the section `Unreleased` will contain all changes that are not released untill now.

Reply:
Resolved in this [commit](https://github.com/MariaMaximova/convert_temp/pull/9/commits/ea07c0df2d274ab51fb7e4346d05465b52c56248)